### ISSUE TITLE
Android - Emoji not showing up correctly after updating Appcompat in older OS

### ISSUE THREAD

**Issue by geriux** [wordpress-mobile/gutenberg-mobile#5429]
**Describe the bug**
We recently [got pinged](https://github.com/wordpress-mobile/WordPress-Android/issues/17565#issuecomment-1403805689) to investigate an issue on Android related to newer emojis not showing up in some scenarios.

I tried updating Appcompat to `1.4.2` and after testing in a few emulators I saw newer emojis weren't showing up for older OS versions of Android.

At first, I thought it was related to our custom implementation with Aztec but this was discarded after I tested the built-in React Native component and found the same issue. I checked that React Native is extending `AppCompatActivity` as it is suggested in this [documentation](https://developer.android.com/develop/ui/views/text-and-emoji/emoji2), I also checked it uses `AppCompatTextView` and `AppCompatEditText` internally.

This issue only happens with the `TextInput` component, for the `Text` component it does work as expected:

<kbd><img width=250 src="https://user-images.githubusercontent.com/4885740/215459482-7d7cdca9-993b-4cf1-b468-941534a923e5.png" /></kbd>
**Emulator running OS 10**

The example above was also done using the latest React Native version `0.71.0`.

We should continue investigations to see if there are other workarounds to be able to support newer emojis in older versions of Android.

**Comment by fluiddot** [comment]
@geriux To better understand the impact of the issue, is this issue affecting `TextInput` and `RichtText`/`Aztect` components? If so, since this will be affecting the content of posts/pages, I think we should prioritize it.

**Comment by geriux** [comment]
> @geriux To better understand the impact of the issue, is this issue affecting TextInput and RichtText/Aztect components? If so, since this will be affecting the content of posts/pages, I think we should prioritize it.

That's correct it is affecting both TextInput and RichText/Aztec components on older versions of Android's OS.

**Comment by SiobhyB** [comment]
I changed the priority from high to medium. I agree this is something we should fix, but think the fact this is limited to older OS version and that we haven't had many reports warrants the lowering of the priority. Open to differing thoughts!
